presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/SchemaDaoUtil.java

```diff
@@ -57,6 +57,23 @@ private static void createTables(SchemaDao dao)
         dao.createTableDeletedShards();
         dao.createTableBuckets();
         dao.createTableShardOrganizerJobs();
+
+        // for upgrading compatibility
+        alterTable(dao::alterTableTablesWithDeltaDelete);
+        alterTable(dao::alterTableTablesWithDeltaCount);
+        alterTable(dao::alterTableShardsWithIsDelta);
+        alterTable(dao::alterTableShardsWithDeltaUuid);
+    }
+
+    private static void alterTable(Runnable alterTableFunction)
+    {
+        try {
+            alterTableFunction.run();
+        }
+        catch (Exception e) {
+            // no-op
+            log.warn(e, "Alter table failed");
```

Comment:
remove

Reply:
changed to another way of logging, since it would be helpful for debug